src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -115,6 +113,13 @@ public void run() {
             throwIfCancelled();
             updateMetadata(module, outputFile);
             throwIfCancelled();
+            if (converter != null) {
```

Comment:
Should be a for loop which iterates on all the converters, rather than a `converter != null` thing...